src/DIRAC/__init__.py

```diff
@@ -88,19 +91,31 @@
 # Importing _strptime before instantiating the threads seem to be a working workaround
 import _strptime
 
-
 # Define Version
+if six.PY3:
+  from pkg_resources import get_distribution, DistributionNotFound
 
-majorVersion = 7
-minorVersion = 2
-patchLevel = 0
-preVersion = 24
-
-version = "v%sr%s" % (majorVersion, minorVersion)
-if patchLevel:
-  version = "%sp%s" % (version, patchLevel)
-if preVersion:
-  version = "%s-pre%s" % (version, preVersion)
```

Comment:
In Python 3 this code now grabs the version data from setuptools, which itself gets it from `git`. In Python 3 this is always PEP-440 style with `vXrYpZ-preN` becoming `X.Y.ZaN`. To make the migration easier the new `DIRAC.__version__` variable is always PEP-440 style.

To allow setuptools to correctly detect the version we should start pushing tags in the new style as well. It's a little ugly but I think it will be easier to double tag commits instead of trying to work around it. I could consider monkey patching setuptools_scm to understand DIRAC's current version (and already have locally) but it's really ugly and liable to randomly breaking when upstream changes are made.

Reply:
no please no Monkey patching.
I think double tag is a low hanging fruit, even more affordable as @petricm progresses with release automation